README.md

```diff
@@ -1,6 +1,6 @@
 # Bulk Data Extract (BDE) Processor
 
-Informix BDE to PostgreSQL converter. A reimplementation of the original
+Informix BDE to PostgreSQL converter. A reimplementation of the legacy
```

Comment:
I would possibly characterise this more as something like:
`Pipeline to extract, transform and load Landonline (Informix) data to the LINZ Data Service (PostgreSQL).`